AirCasting/Onboarding/NearAirDescription.swift

```diff
@@ -0,0 +1,75 @@
+// Created by Lunar on 09/06/2021.
+//
+
+import SwiftUI
+
+struct NearAirDescription: View {
+    var completion: () -> Void
+    var body: some View {
+        ScrollView {
```

Comment:
It has a reason. When we use Scroll View we are able to use horizontal view of the phone, otherwise we won't cover every content placed on the screen. Can we stay with it please 😇?

Reply:
I'm not convinced 🙃 I'm calling @mateuszmackowiak & @pawelgil  to this discussion 🕵🏻‍♂️

It turns out I can show you what I mean by "sth bad can happen". I have a small iPhone and those two views with scroll view look like this (screenshots). 
And the second thing is that it took me a second to realize I can scroll, because this view doesn't suggest it's scrollable, it's just a simple view with the button. So my first thought was that there's sth wrong with padding ;) 

![IMG_4345](https://user-images.githubusercontent.com/46443841/123610662-14c5b400-d801-11eb-9408-a6212fa1857a.PNG)


![IMG_4344](https://user-images.githubusercontent.com/46443841/123610647-10999680-d801-11eb-9034-793ae8298e07.PNG)